app/assets/javascripts/fae/form/_validator.js

```diff
@@ -51,6 +51,13 @@ Fae.form.validator = {
       if ($(".field_with_errors").length) {
         $('.alert').slideDown('fast').removeClass('hide').delay(3000).slideUp('fast');
       }
+
+      if ($('.invalid').length) {
```

Comment:
@JuliannaGreen I have a couple concerns with this change:
I don't think we should be sniffing something so general as an `invalid` class. There are cases where we have multiple forms on the page (as with nested forms) which may be prevented from submitting if the parent form has errors. If this class is reliable at determining the validity of the form, then maybe scoping the lookup to the form would be better?

If we can't count on the `is_valid` flag, likely that's where the bug is. I believe it's only purpose is to determine whether or not to submit the form, so you're method here would likely mean stripping out the flag altogether. However, I would bet the issue is in properly setting the flag during the `_judgeIt()` call.